p2pool/src/server/p2p/network.rs

```diff
@@ -2859,8 +3487,15 @@ where S: ShareChain
 
     pub async fn dial_seed_peers(&mut self) -> Result<(), Error> {
         info!(target: LOG_TARGET, "Dialing seed peers...");
-        let seed_peers = self.parse_seed_peers().await?;
-        self.join_seed_peers(seed_peers).await?;
+        match self.parse_seed_peers().await {
+            Ok(seed_peers) => {
+                self.join_seed_peers(seed_peers).await;
+            },
+            Err(e) => {
+                warn!(target: LOG_TARGET, "Failed to parse seed peers: {e:?}");
+                return Err(e);
+            },
+        }
```

Comment:
why this change?
If you just want to print out the log, you could use inspect_err

Reply:
Function `async fn join_seed_peers` previously returned an empty result when in fact it was infallible. It was changed to not return anything, hence the change here, with the addition of the log message.

Reply:
talking about `match self.parse_seed_peers().await`